db/schema.rb

```diff
@@ -43,8 +43,6 @@
     t.boolean  "fogged"
   end
 
-  add_index "checkins", ["device_id"], name: "index_checkins_on_device_id", using: :btree
```

Comment:
just ran `rake db:migrate` and nothing is changed, I went digging and found the commit to come from the dev branch i merged before pushing here.